Signal/src/ViewControllers/OWSTableViewController.m

```diff
@@ -196,6 +197,20 @@ - (void)loadView
     [self.tableView registerClass:[UITableViewCell class] forCellReuseIdentifier:kOWSTableCellIdentifier];
 }
 
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+
+    [self.navigationController.navigationBar setTranslucent:NO];
+}
+
+- (void)viewWillAppear:(BOOL)animated
+{
+    [super viewWillAppear:animated];
+
+    self.tableView.tableFooterView = [[UIView alloc] initWithFrame:CGRectZero];
```

Comment:
just curious, why do we always want a footer view? I realize it's sized zero so it's not going to appear, but it seems like you did this for a reason. Why?